### DISCUSSION TITLE
Linear Advance

### DISCUSSION THREAD

**Discussion by o0LINNY0o** [Jyers/Marlin#2010] [General Discussion]
Starting to see marlin releases with linear advance. Is there any plans to reignite jyers ? 

Don't really like the feel of professional since it feels jerky. And the excellent work from tititopher ? needs a screen flash as well. 

But I feel Linear Advance is what will make me move . Especially if the v2 4.2.2. boards don't need to be modified

**Comment by Tapap84** [comment]
tititopher has done amazing work with jyers and is releasing builds on discord. the group is in french. go to ender 3 firmware tab and youll find it. his 3.0.2b has linear advance, hes had it for a while. https://discord.gg/neFzHyhh
